packages/gatsby-react-router-scroll/src/ScrollBehaviorContext.ts

```diff
@@ -47,16 +69,22 @@ class ScrollContext extends React.Component {
     this.scrollBehavior.updateScroll(prevRouterProps, { history, location })
   }
 
-  componentWillUnmount() {
+  componentWillUnmount(): void {
+    // Type definitions from scroll-behavior forgot to include this method
+    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+    // @ts-ignore
```

Comment:
Is there a PR or issue documenting this on their side?

Reply:
[Merged](https://github.com/taion/scroll-behavior/pull/365#issue-386502731)

Reply:
Great. Can we bump the dependency to `0.10.0` so that we get those types in